src/Sylius/Bundle/AdminBundle/Resources/config/grids/order.yml

```diff
@@ -85,6 +85,11 @@ sylius_grid:
                     label: sylius.ui.date
                     options:
                         field: checkoutCompletedAt
+                channel:
+                    type: entity
+                    label: sylius.ui.channel
+                    options:
+                        class: Sylius\Component\Core\Model\Channel
```

Comment:
We should use a parameter here ``%sylius.model.channel.class%``, this way it will still work after someone overrides channel model. ;)